a3p-integration/proposals/a:upgrade-next/prepare-test-env-ava.js

```diff
@@ -0,0 +1,7 @@
+import '@endo/init/debug.js';
```

Comment:
I don't see any reason for upgrade tests to run under lockdown. They're scripting the chain which is fully locked down. Please leave these as is so we don't have this extra boilerplate to write the tests.